pkg/operator/staticpod/controllers.go

```diff
@@ -79,6 +80,9 @@ type staticPodOperatorControllerBuilder struct {
 	guardCreateConditionalFunc    func() (bool, bool, error)
 
 	revisionControllerPrecondition revisioncontroller.PreconditionFunc
+
+	// node information
```

Comment:
same comment applies here

Reply:
Oh, this was just following the convention in the other groupings in this struct, but since it's also one un-exported variable, I'll just remove the comment